### ISSUE TITLE
Transforming to LLVM IR

### ISSUE THREAD

**Issue by edg-l** [raviqqe/melior#365]
Do you know if its possible to use mlir-translate programatically within rust?

What i want to know is how to convert MLIR, all already transformed to llvm dialect, to llvm ir for use within the llvm api, e.g to compile to assembly, without using any tool like mlir-translate.

**Comment by edg-l** [comment]
I opened this pr on llvm https://github.com/llvm/llvm-project/pull/73117

**Comment by raviqqe** [comment]
Yeah, we can definitely add the feature when the LLVM side supports it.

**Comment by edg-l** [comment]
The pr got merged, so this will probably be in llvm 18

**Comment by gavrilikhin-d** [comment]
So is it supported by melior now?

**Comment by raviqqe** [comment]
No, I don't think we made the change yet.
